sw/device/lib/testing/clkmgr_testutils.h

```diff
@@ -20,8 +21,9 @@
 inline bool clkmgr_testutils_get_trans_clock_status(
```

Comment:
should this be static? or is it used outside this file too?

Reply:
I made it static.